<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="scheme">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>22</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Схема добавления:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_scheme">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>18</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="numwell">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>74</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер скважины:</string>
   </property>
  </widget>
  <widget class="QLabel" name="numport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер порта:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_numwell">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>70</y>
     <width>130</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_numport">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>120</y>
     <width>130</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lenS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Полудлины прдзаданного пути:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lenleft">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>178</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lenright">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>178</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="left">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>208</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Левая</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="right">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>207</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Правая</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
